build/linux/installer/conf/td-agent-bit.conf

```diff
@@ -16,7 +16,7 @@
     Tag oms.container.log.la.*
     Path ${AZMON_LOG_TAIL_PATH}
     Read_from_Head true
-    DB /var/log/omsagent-fblogs.db
+    DB /var/log/ama-logs-fblogs.db
```

Comment:
This will be an issue meaning name change causes new DB file which will cause the all the previous DB state associated to the log files on the existing node when the agent restarted. We will have to keep old DB name to avoid any such regression.